### ISSUE TITLE
SSL certificate of the Pfam webpage has expired

### ISSUE THREAD

**Issue by TlaskalV** [AstrobioMike/GToTree#28]
Hello Mike,
when trying new conda installation of GToTree I was not able to finish `gtt-test.sh` command. It gives me downloading error for one of the models 

```
The problem child was PF00238.19.
```
When trying manual `curl` of the model, I got
```
curl: (60) SSL certificate problem: certificate has expired  
```
It seems that referenced webpage can not be accessed. 
https://github.com/AstrobioMike/GToTree/blob/1ac6b35dfaf45ff81c288c12541049938b14bca8/bin/gtt-pfam-search#L368

Do you know how to heck the security settings of the `curl`? Maybe `-k/--insecure` option?

Thanks!

**Comment by TlaskalV** [comment]
Thanks for fast commit. Indeed, it expired yesterday according to my browser. Let's see if the Pfam support will renew it.

**Comment by AstrobioMike** [comment]
Beautiful :) Thanks so much for not only taking your time to tell me about this, but to also completely troubleshoot it!

The pfam certificate is out of date by 2 days, i'm sure they'll catch up with it soon, but can't really wait on it, so i've added the `--insecure` flag like you noted to the places where that is called (it happens in the main GToTree script and the `gtt-pfam-search` you pointed to depending on if it's called by itself with `gtt-pfam-search` or within a regularly `GToTree` run – this is not sensible of course, just a consequence of me still being a messy, nascent "programmer", haha). Changes in https://github.com/AstrobioMike/GToTree/commit/7a307a87260d5a90c15707bfce32734540c47093

It's implemented in version 1.5.39, which is currently building and uploading to conda, so should be available there within about 10 minutes

Thanks again so much for your time and info, @TlaskalV! 
![image](https://user-images.githubusercontent.com/13923308/100553851-6abf8280-3245-11eb-849e-d354c23ca38b.png)
